mitmproxy/flow/export.py

```diff
@@ -4,23 +4,34 @@
 import re
 from textwrap import dedent
 
-from six.moves.urllib.parse import quote, quote_plus
+import six
+from six.moves import urllib
 
 import netlib.http
 
 
+def _native(s):
```

Comment:
The `_native` function is used in the `dicstr` function to convert a dictionary to string (sort of like json.dumps), so for eg. [this params dict](https://github.com/mitmproxy/mitmproxy/blob/master/test/mitmproxy/data/test_flow_export/python_get.py#L11) has a list as value.